src/Control/Email/Email.php

```diff
@@ -256,10 +257,13 @@ public function __construct(
     }
 
     /**
+     * @deprecated 4.12.0 Class will extend Symfony\Component\Mime\Email
```

Comment:
```suggestion
     * @deprecated 4.12.0 Will be removed without equivalent functionality
```
This lines up with what we're saying in `Debug()` and is more directly useful information.

Reply:
Updated